### ISSUE TITLE
prefer-primordials: Don't use `in` operator, use `ObjectHasOwn`

### ISSUE THREAD

**Issue by petamoriken** [denoland/deno_lint#993]
related: https://github.com/denoland/deno_lint/issues/986#issuecomment-1003818902

### Lint Name

prefer-primordials

### Code Snippet

```ts
"a" in b;
```

### Expected Result

Suggest using `ObjectHasOwn`.

```ts
ObjectHasOwn(b, "a");
```

### Actual Result

Nothing.

### Version

deno 1.17.1 (release, aarch64-apple-darwin)
v8 9.7.106.15
typescript 4.5.2

**Comment by malectro** [comment]
Would this be a good first issue to take on? Looks simple, and I'd like to try contributing.

**Comment by bartlomieju** [comment]
@malectro yes, this is a good first issue. See https://github.com/denoland/deno_lint/pull/995 and https://github.com/denoland/deno_lint/pull/992 for examples on how to implement that.
